src/index.html

```diff
@@ -7,12 +7,53 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
+    />
+    <link
+      rel="stylessheet"
+      href="/src/styles/index.css"
```

Comment:
The `href` attribute should use a relative path without `/src` since the path is relative to the `index.html` file which is already in the `src` directory. The correct path would be `styles/index.css`.